src/layouts/MainLayout.vue

```diff
@@ -168,26 +168,64 @@ export default {
       console.error(err)
     }
 
-    // const lastReceived = this.lastReceived
-    const t0 = performance.now()
-    const refreshMessages = () => {
-      // Wait for a connected electrum client
-      if (!this.$electrum.connected) {
-        setTimeout(refreshMessages, 100)
-        return
+    // Start wallet initialization
```

Comment:
This does not fix the problem with electrum servers disconnecting randomly later on after the app is started. I put that stuff on `setElectrumClient` for that reason -- because the subscriptions need to be remade every time you reconnect.

What needs to be done is only calling the electrum client connect stuff AFTER xPriv is set.